pkg/proxy/apis/config/validation/validation.go

```diff
@@ -276,8 +276,7 @@ func validateKubeProxyNodePortAddress(nodePortAddresses []string, fldPath *field
 
 	for i := range nodePortAddresses {
 		if _, _, err := net.ParseCIDR(nodePortAddresses[i]); err != nil {
-			allErrs = append(allErrs, field.Invalid(fldPath, nodePortAddresses, "must be a valid IP block"))
-			break
+			allErrs = append(allErrs, field.Invalid(fldPath, nodePortAddresses[i], "must be a valid CIDR"))
```

Comment:
Shouldn't this be `fldPath.Index(i)` ?

Reply:
I think this needs to remain fldPath because nodePortAddresses, will not return the full CIDR and it splits after the colon.

Ex. with the change 
```
   validation_test.go:965: Expected error: KubeProxyConfiguration.NodePortAddresses: Invalid value: "::0": must be a valid CIDR, got KubeProxyConfiguration.NodePortAddresses[0]: Invalid value: "::0": must be a valid CIDR
```

Reply:
I'm sorry, I don't understand your reply.  The test is failing because the literal error it is expecting references the field "NodePortAddresses", but we almost always want to flag WHICH address so it should be "NodePortAddresses[0]".

Reply:
That's true, I totally read this incorrectly in my test output, fixing now